packages/dds/tree/src/test/shared-tree/editing.spec.ts

```diff
@@ -1764,7 +1764,7 @@ describe("Editing", () => {
 			);
 		});
 
-		it.skip("concurrent cycle creating move", () => {
+		it("concurrent cycle creating move", () => {
```

Comment:
L1793 needs an update, probably just removal.

Does the merge outcome still produce a cycle in the tree?

Reply:
I haven't investigated further, but the test does check that the final document is empty even though there were only move edits, so I assume the behavior merge hasn't changed.